testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1378,16 +1380,18 @@ QUnit.module('Lookup', {
 
 QUnit.module('label integration', () => {
     QUnit.test('lookup should pass containerWidth equal to field width', function(assert) {
-        const constructorMock = sinon.stub();
-        Lookup.mockTextEditorLabel(constructorMock);
+        this.TextEditorLabelMock = (args) => { this.labelArgs = args; return new TextEditorLabel(args); };
```

Comment:
seems like we can write smth like 
```suggestion
        this.TextEditorLabelMock = sinon.spy((args) =>  new TextEditorLabel(args));
```

and get rid of `this.labelArgs`

Reply:
in this test we check label args